db/schema.rb

```diff
@@ -50,7 +50,7 @@
     t.string "name"
     t.string "photo"
     t.text "bio"
-    t.integer "posts_counter"
+    t.integer "posts_counter", default: 0
   end
 
   add_foreign_key "comments", "posts"
```

Comment:
- [ ] Your migrations fail after running `rails db:migrate` command. Kindly fix those errors so that we can be able to review your project.

<img width="659" alt="image" src="https://user-images.githubusercontent.com/77190203/194604764-8fae5aff-e550-4a3f-8184-5263cecb66c8.png">

Notice:
To ensure that your migrations work as expected, I recommend deleting the `schema.rb` file, and then running `rails db: reset`. After that, run `rails db:migrate`. If the migrations do not show any errors, that's fine, you may request a new review. But if the migrations show errors, try to solve those errors and repeat the steps.